### PR TITLE
ask to automatically add Deno to the PATH

### PR DESCRIPTION
**Pull request by SirJosh3917**
This modification to the script will ask the user if they want to automatically add Deno to their PATH.

### CODE REVIEW

**Comment by brkeejp** [comment]
very useful fork. Even I wonder same thing. I'm adding my code to ~/.profile btw.
`echo '

export DENO_INSTALL="$HOME/.deno";
export PATH="$DENO_INSTALL/bin:$PATH"' >> $HOME/.profile`

File: install.sh [hunk 1]
```diff
@@ -60,8 +60,22 @@ else
...
+	# shellcheck disable=SC2039
+	read -p "Install Deno into your PATH? [y/N]: " -r REPLY || REPLY="N"
+
+	if [ "$REPLY" = "Y" ] || [ "$REPLY" = "y" ]; then
+		{
+			echo ""
+			echo "# automatically added by deno_install"
+			echo "export DENO_INSTALL=\"$deno_install\""
+			echo "export PATH=\"\$DENO_INSTALL/bin:\$PATH\""
+		} >>"$HOME"/"$shell_profile"
```
Comment (brkeejp) [inline install.sh:73]:
```suggestion
		echo '
		
		# automatically added by deno_install
		export DENO_INSTALL="$deno_install";
		export PATH="\$DENO_INSTALL/bin:\$PATH";
		' >> \$HOME/$shell_profile
```

**Review by brkeejp — APPROVED** [review]
Worked for me :) I hope that's not get complicated :)

**Review by brkeejp — APPROVED** [review]

File: install.sh [hunk 2]
```diff
@@ -60,8 +60,22 @@ else
...
+	# shellcheck disable=SC2039
+	read -p "Install Deno into your PATH? [y/N]: " -r REPLY || REPLY="N"
+
+	if [ "$REPLY" = "Y" ] || [ "$REPLY" = "y" ]; then
+		echo '
+		
+		# automatically added by deno_install
+		export DENO_INSTALL="'"$deno_install"'";
+		export PATH="'"$"'DENO_INSTALL/bin:'"$"'PATH";
+		' >>"$HOME/$shell_profile"
```
Comment (tokiedokie) [inline install.sh:73]: What is `$shell_profile`?

Comment (SirJosh3917) [reply install.sh:73]: Defined in lines 59-62
```shell
case $SHELL in
/bin/zsh) shell_profile=".zshrc" ;;
*) shell_profile=".bash_profile" ;;
esac
```

**Comment by ry** [comment]
Thanks for the patch, but we're trying to keep the script simple https://github.com/denoland/deno_install/blob/4d960274a86fa46265ca5ba9adc01a2405baded1/install.sh#L3
Closing without merge.

**Pull request closed without merging**